src/tools/kdb/cache.hpp

```diff
@@ -25,17 +25,20 @@ class CacheCommand : public Command
 
 	virtual std::string getSynopsis () override
 	{
-		return "{enable,disable,clear}";
+		return "{enable,disable,default,clear}";
 	}
 
 	virtual std::string getShortHelpText () override
 	{
-		return "Enable, disable or clear the cache.";
+		return "Enable, disable, clear the cache or revert to default.";
 	}
 
 	virtual std::string getLongHelpText () override
 	{
-		return "";
+		return "This command is used to enable or disable the cache and to revert\n"
```

Comment:
Afaik the getLongHelpText is no longer used. So it is better to leave it empty (or even better: remove the whole function in all the classes).

Reply:
My `kdb` tool displays the long help text. I can remove it in all classes, but likely not today.

Many tools do have some information in the long help text. Do we just throw that out, or is the idea to merge everything into one single help text, be it long or short?

Reply:
Yes, you are right. The text is shown when typing in wrong arguments or similar.

So then leave it as is, as it would be a lot of work to unify all these texts and there would be only little gain.